### DISCUSSION TITLE
JFreeChart 1.54 or 1.60 ?

### DISCUSSION THREAD

**Discussion by vsonnier** [jfree/jfreechart#278] [General]
Hello @jfree thanks for all your work, and support over the years. 

Especially thanks @Yuri-Blankenstein-TNO for all your recent additions : https://github.com/jfree/jfreechart/pulls?q=is%3Apr+author%3AYuri-Blankenstein-TNO

I'm particulary interested by the offline rendering, but the other additions looks cool too !

Speaking of which, I've read somwhere that @jfree was hesitant to release a v1.54 given the amount of changes.

Whatever you decide to name it, pleeeeeaaaaase keep the Java 1.8 minimum requirement :)

Anyway, I'm looking forward to a new release with all these features.

Regards,

Vincent

**Comment by trashgod** [comment]
Issue #203 suggests a possible roadmap saying, "anyone wanting to remain on Java 8 could still use the JFreeChart 1.5.x versions." Some recent back ports are identified [here](https://stackoverflow.com/search?q=user%3A230513+%5Bjfreechart%5D+1.5.4).

**Comment by Yuri-Blankenstein-TNO** [comment]
Thank you @vsonnier for starting this thread. The [Eclipse TRACE4CPS](https://projects.eclipse.org/projects/technology.trace4cps) project will benefit a lot when a new Java8 release of JFreeChart is made, as the Eclipse Foundation approved the usage of JFreeChart for Eclipse TRACE4CPS, see [CQ 23674](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23674).

Considering the comments of @jfree it might be advisable to create two releases, i.e. a 1.5.4 maintenance release and a 1.6.0 release that contains some (API) additons, also see https://github.com/jfree/jfreechart/pull/220#issuecomment-1008942286. Currently I've prepared a [JFreeChart 1.6.0-RC1 pre-release](https://github.com/Yuri-Blankenstein-TNO/jfreechart/releases/tag/v1.6.0_RC1) in my fork. I've synced its built (but unsigned) binaries to the Eclipse TRACE4CPS git repository, see https://gitlab.eclipse.org/eclipse/trace4cps/trace4cps/-/merge_requests/1.

**Reply by Yuri-Blankenstein-TNO** [reply]
@jfree are there any plans yet on creating these new releases? While awaiting these releases I asked permission to the Eclipse Foundation EMO team to use my fork for Eclipse TRACE4CPS, see https://gitlab.eclipse.org/eclipse/trace4cps/trace4cps/-/issues/1#note_775265

**Comment by ssathish062087** [comment]
Hi Team,

we are looking forward to 1.5.4 maintenance release that is compatible with Java8 without API changes. Can you provide tentative timeline for release availability.

Regards,
S sathish
